code/modules/mob/living/simple_animal/hostile/bear.dm

```diff
@@ -99,8 +99,57 @@
 		to_chat(user, "<span class='info'>You strap the armor plating to [A] and sharpen [A.p_their()] claws with the nail filer. This was a great idea.</span>")
 		qdel(src)
 
+mob/living/simple_animal/hostile/bear/butter //The mighty companion to Cak. Several functions used from it.
+	name = "Kerrygold"
```

Comment:
Change the name to something that's not exactly the trademark name please

Reply:
```suggestion
	name = "Terrygold"
```

Reply:
Good suggestion MrDoomBring! I'll merge it.